CONTRIBUTING.md

```diff
@@ -61,7 +61,12 @@ If you wish to build a copy of Babel for distribution, then run:
 $ make build-dist
 ```
 
-and access the files from `packages/babel-core/dist`.
+and access the built files for individual packages from `packages/<package-name>/lib`.
+
+The "top-level" package is `babel-cli`.  Accordingly, its built files live in `packages/babel-cli/lib`.
```

Comment:
Maybe it could go better as part of https://github.com/babel/babel#packages in the readme or the individual readme of babel-cli?